### DISCUSSION TITLE
Welcome to firely-net-sdk Discussions!

### DISCUSSION THREAD

**Discussion by marcovisserFurore** [FirelyTeam/firely-net-sdk#1562] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MaryTWelch** [comment]
Hi all!  I'm hoping to make sure I understand the FhirDateTime accurately as we convert it to the .NET System Date time.  Is this an accurate way to do this?  Is it the best way to do this?

![image](https://user-images.githubusercontent.com/61983738/104954836-b281a100-598e-11eb-81d9-b23419256bd4.png)


Thanks for your help.

Mary

**Reply by MaryTWelch** [reply]
To add to this, all our internal dates are stored in UTC.

**Reply by marcovisserFurore** [reply]
Hi @MaryTWelch, 
Congratulations! You are the first who enters a discussion item on this new discussion board.

What you are doing here (from `FhirDateTime` to .NET `System.DateTime`) is correct. The last line (`sysDtTm.UtcDateTime`) is not really necessary because to the `System.TimeSpan.Zero` is already doing that. So the code could be a bit shorter:

```c#
if (condition.Onset is FhirDateTime fdt)
{
   info.OnsetDate = fdt.ToDateTimeOffset(System.TimeSpan.Zero);
}
```

**Reply by MaryTWelch** [reply]
@marcovisserFurore thanks for the help!  When I try to shorten the statement as you mention, I get the following.  I need a DateTime Object returned specifically, this would return a DateTimeOffset object.
![image](https://user-images.githubusercontent.com/61983738/105343447-42019c80-5ba7-11eb-942b-730ac15bcdc9.png)

**Reply by MaryTWelch** [reply]
Actually I think I just found an answer in your code base, in the ElementNavFhirExtensions.cs class.  Thanks again!!
![image](https://user-images.githubusercontent.com/61983738/105344565-b6890b00-5ba8-11eb-85d9-e92edf814238.png)

**Reply by marcovisserFurore** [reply]
I wrote my lines without a IDE, so I was not corrected for any syntax errors ;)

I am happy that you solved it!
